numba_rvsdg/core/datastructures/scfg.py

```diff
@@ -121,7 +151,11 @@ def find_head(self) -> str:
 
     def compute_scc(self) -> List[Set[str]]:
         """
-        Strongly-connected component for detecting loops.
+            Computes the strongly connected components (SCC) of the control 
```

Comment:
Mention here that this uses Tarjan's algorithm. https://en.wikipedia.org/wiki/Tarjan%27s_strongly_connected_components_algorithm

Reply:
Resolved at [9eed76e](https://github.com/numba/numba-rvsdg/pull/59/commits/9eed76e4e286a1823a6602b9bf538e5a3c648c50)